### PR TITLE
added license

### PR DESCRIPTION
**Pull request by ritikk112**
Added license for
```
Introduction to Embedded Systems, Second Edition](https://ptolemy.berkeley.edu/books/leeseshia/releases/LeeSeshia_DigitalV2_2.pdf) - Edward Ashford Lee, Sanjit Arunkumar Seshia (PDF)
```

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description

Add License Notes to the following books
The book is present in books/free-programming-books-subjects.md

[Introduction to Embedded Systems, Second Edition](https://ptolemy.berkeley.edu/books/leeseshia/releases/LeeSeshia_DigitalV2_2.pdf) - Edward Ashford Lee, Sanjit Arunkumar Seshia (PDF)

License Line found on Page 2 of the PDF.

Added (CC BY-NC-ND) "

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 * [Control and Embedded Systems](http://www.learn-c.com) (HTML)
 * [Discovering the STM32 Microcontroller](http://www.cs.indiana.edu/~geobrown/book.pdf) (PDF) (CC BY-NC-SA)
 * [First Steps with Embedded Systems](https://www.phaedsys.com/principals/bytecraft/bytecraftdata/bcfirststeps.pdf) - Byte Craft Limited (PDF)
-* [Introduction to Embedded Systems, Second Edition](https://ptolemy.berkeley.edu/books/leeseshia/releases/LeeSeshia_DigitalV2_2.pdf) - Edward Ashford Lee, Sanjit Arunkumar Seshia (PDF)
+* [Introduction to Embedded Systems, Second Edition](https://ptolemy.berkeley.edu/books/leeseshia/releases/LeeSeshia_DigitalV2_2.pdf) - Edward Ashford Lee, Sanjit Arunkumar Seshia (PDF) (CC EY-NC-ND)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:311]: * EY -> BY
* Please edit your PR description to indicate how/where you found the license notation

Comment (ritikk112) [reply books/free-programming-books-subjects.md:311]: Updated the description of the PR, can you do a quick verification?

Comment (eshellman) [reply books/free-programming-books-subjects.md:311]: still wrong

Comment (ritikk112) [reply books/free-programming-books-subjects.md]: Updated description and edited license for the book, can you do a quick review again?

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**